test/test-pages/tumblr/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Minecraft 1.8 - The Bountiful Update - Minecraft 1.8 - The Bountiful Update",
```

Comment:
Hm, odd - does tumblr always repeat the title? It looks odd, but I can see this was already in the original page...